server/src/main/java/org/opensearch/index/SegmentReplicationPressureService.java

```diff
@@ -60,7 +60,7 @@ public class SegmentReplicationPressureService implements Closeable {
 
     public static final Setting<Integer> MAX_INDEXING_CHECKPOINTS = Setting.intSetting(
         "segrep.pressure.checkpoint.limit",
-        4,
+        10,
```

Comment:
increasing to 10, allows segrep lag of 100 secs assuming 10 sec refresh interval but the `segrep.pressure.time.limit` is 300secs for backpressure. Just for my understanding, why do we want to keep this 200sec gap still?

Reply:
Made it 30 now.